Resources/Prototypes/Recipes/Cooking/meal_recipes.yml

```
@@ -590,6 +590,16 @@
     FoodDonkpocketWarm: 3
     FoodCheeseSlice: 1
     FoodTomato: 1
+- type: microwaveMealRecipe
```

```suggestion
    FoodTomato: 1
    
- type: microwaveMealRecipe
```
Should have a line after the previous recipe.